TTHAnalysis/python/tools/higgsRecoTTH_NoTopTag.py

```diff
@@ -0,0 +1,172 @@
+from __future__ import division
```

Comment:
Please condense the module into a single higgsRecoTTH.py module

Reply:
working on it